README.md

```diff
@@ -1,2 +1,242 @@
-# release-notes-presence-check
-A GH action for validating presence of release notes.
+# Release Notes Presence Checker
+
+A GH action for validating the presence of release notes in pull requests.
+
+## Motivation
+
+This action is designed to help maintainers and contributors ensure that release notes are present in pull requests. It can be used to prevent common issues such as:
+- Missing release notes
+- Incomplete release notes
+- Incorrect release note formats
+- Release notes in the wrong location
+
+## Requirements
+- **GitHub Token**: A GitHub token with permission to fetch repository data such as Issues and Pull Requests.
+- **Python 3.11+**: Ensure you have Python 3.11 installed on your system.
+
+## Inputs
+
+### `GITHUB_TOKEN`
+- **Description**: Your GitHub token for authentication. Store it as a secret and reference it in the workflow file as secrets.GITHUB_TOKEN.
+- **Required**: Yes
+
+### `pr-number`
+- **Description**: The pull request number to check for release notes. Example: `1`.
+- **Required**: Yes
+
+### `github-repository`
+- **Description**: The GitHub repository to check for version tags. Example: `AbsaOSS/release-notes-presence-check`.
+- **Required**: Yes
+
+### `location`
+- **Description**: The location of the release notes in the pull request. Example: `body`.
+- **Required**: No
+
+### `title`
+- **Description**: The title of the release notes in the pull request. Example without regex: `Release Notes:`, with regex: `[Rr]elease [Nn]otes:`.
+- **Required**: No
+
+### `skip-labels`
```

Comment:
Based on yaml, there is default as well.

Reply:
Fixed in commit